### ISSUE TITLE
Conditions

### ISSUE THREAD

**Issue by markbrough** [pwyf/2016-index-data-quality-tests#38]
**Description**
The terms and conditions of the activity may also be referred to as benchmarks, priors, or involve words such as “subject to…”. They are specific to an individual activity and explain what the recipient must do in order to be eligible for the funds to be released. This is only expected if the activity is at least in the implementation phase.

**Existing tests**

```
conditions exists (if activity-status/@code is at least 2)?
document-link/category[@code='A04'] exists (if activity-status/@code is at least 2)?
```

**Issues**
No issues have been identified.

**2016 Aid Transparency Index test**
We are not planning to change this test in 2016.

**Comment by wendyrogers** [comment]
If there are no condition for an activity do you still require that conditions is reported but with a zero to indicate that there are no conditions to add <conditions attached="0"/ > ?

**Comment by markbrough** [comment]
@iatisupport yes - there are two tests, conditions data and conditions documents. For the first test, conditions data must be reported for all activities in the implementation stage or higher; if there are no specific conditions attached, you can just do:

``` xml
<conditions attached="0" />
```

**Comment by cmmarschner** [comment]
MCC publishes details around our administrative funding.  We don’t have results or conditions or other types of data on this because its not program funding, its administrative. Highly recommend that there is some way to filter out administrative funds from purely programmatic funds - many of the fields you score are only relevant to pure program funds.  That being said, the administrative funds add relevant context so we wouldn't want to exclude them.  This comment applies to numerous fields but I'm only posting it here, since it is relevant to the previous comments.

**Comment by cmmarschner** [comment]
Could we exclude, or test differently, lines coded as DAC code administrative funding (91010) as a way of separating out administrative funding?

**Comment by markbrough** [comment]
Thanks for all of the comments.

**For the 2016 Aid Transparency Index**

This test will remain the same as in previous years.

_Notes_
- @cmmarschner suggests an interesting way of separating out administrative funding. However, there are several issues that this approach immediately raises
  - there is no definition of 91010 "Administrative costs" in the [Sector](http://iatistandard.org/201/codelists/Sector/) codelist; the [Aid Type](http://iatistandard.org/201/codelists/AidType/) codelist states that administrative costs should be coded under G01 only if they are "not already included under other ODA items as an integral part of the costs of delivering or implementing the aid provided" which suggests that coding with 91010 might need to be more widely agreed first before the Index encourages such an approach
  - it is not immediately clear which tests this should apply to
  - we are not in a position to understand any adverse effects that this approach could cause to particular donors and would like to be able to discuss and test further before making such a change to a large number of tests
- we would like to discuss this further and will consider this approach for the Index in future years
